.github/workflows/charm-release.yaml

```diff
@@ -0,0 +1,100 @@
+name: Release Charm
+
+on:
+  workflow_call:
+    inputs:
+      artifact:
+        description: "Name of artifact to download before building. Must contain the file artifact.tar.gz."
+        default: ''
+        required: false
+        type: string
+      channel:
+        description: |
+          The channel to which charm should be released. Edge channel is default.
+        default: latest/edge
+        required: false
+        type: string
+      upload-image:
+        description: |
+          Set to true if you want to update the OCI image
+        default: false
+        required: false
+        type: boolean
+      python-version-unit:
+        required: true
+        description: "Python version"
+        type: string
+        default: "['3.8', '3.10']"
+      python-version-func:
+        required: true
+        description: "Python version"
+        type: string
+        default: "3.10"
+      tox-version:
+        required: false
+        description: "Tox version, which should be used, e.g. `<4`"
+        type: string
+        default: ""
+      working-directory:
+        required: false
+        description: "To change working directory"
+        type: string
+        default: "."
+      commands:
+        required: false
+        description: |
+          Command to run functional test in strategies. It defaults to
+          `make functional`, but is defined as a list, so `['make functional']`.
+          This allows you to run multiple tests with different parameters.
+          Examples:
+          - ['FUNC_ARGS="--series jammy" make functional',
+             'FUNC_ARGS="--series focal" make functional']
+          - ['tox -e func -- -sv --series jammy', 'tox -e func -- -sv --series focal']
+        type: string
+        default: "['make functional']"
```

Comment:
Hi @gabrielcocenza, sorry I did not saw this. Can we remove default value here so we will use same approach like in pr.yaml. Thanks

Reply:
See #18

Reply:
Thank you :+1: